node/examples/electron/rnnoise/README.md

```diff
@@ -0,0 +1,41 @@
+# An Electron.js example of Noise Suppression using webnn-native
```

Comment:
Nit, it'd be better to specific it's for RNNoise, e.g. Noise Suppression (RNNoise)

Reply:
Done.